packages/web/src/builders/package/package.impl.ts

```diff
@@ -90,6 +91,9 @@ export function run(
         sourceRoot
       );
 
+      // Delete output path before bundling
+      deleteOutputDir(context.workspaceRoot, options.outputPath);
```

Comment:
Can you bring the `deleteOutputDir` function in this file instead?

The `third-party` folder was copied from the Angular codebase when we removed them as a depenency of `@nrwl/web`. We'll eventually get rid of this folder when we have time to do some refactoring.